packages/stacked_tools/test/services_tests/template_service_test.dart

```diff
@@ -3,14 +3,18 @@ import 'package:stacked_tools/src/constants/message_constants.dart';
 import 'package:stacked_tools/src/exceptions/invalid_stacked_structure_exception.dart';
 import 'package:stacked_tools/src/locator.dart';
 import 'package:stacked_tools/src/models/template_models.dart';
-import 'package:stacked_tools/src/services/template_service.dart';
+import 'package:stacked_tools/src/services/render_template_service.dart';
+import 'package:stacked_tools/src/services/revert_template_service.dart';
+import 'package:stacked_tools/src/services/template_service_utils.dart';
 import 'package:stacked_tools/src/templates/compiled_template_map.dart';
 import 'package:stacked_tools/src/templates/template_constants.dart';
 import 'package:test/test.dart';
 
 import '../helpers/test_helpers.dart';
 
-TemplateService _getService() => TemplateService();
+TemplateServiceUtils _getUtilsService() => TemplateServiceUtils();
```

Comment:
We shouldn't have multiple service's test in the same file. These need to be different testing files.